.kokoro/publish-docs.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+cd $(dirname $0)/..
+
+npm install
+
+npm run docs
+
+# Publish documentation with docuploader.
+python3.6 -m pip install gcp-docuploader
+
+python3.6 -m docuploader create-metadata \
+			--name pubsub \
+			--version 0.27.0 \
```

Comment:
Is this the version of the npm module?  That feels like a scary thing to have hard-coded in here.

Reply:
Sorry for the scare, I'm using this branch to iron out any weirdness before creating a PR to update synthtool templates. I will fetch the version (and the name) from the appropriate place. What's the best way to get the npm module version?

Reply:
Assuming the version has been published to npm - you can run:
```
npm view @google-cloud/pubsub version
```

otherwise, I think you'd have to parse `package.json`, and read the `version` property.  @bcoe do you know if there's a way to fetch the current version from `package.json` with the `npm` cli?

Reply:
I've tended to do something like this if I'm getting really creative:

`curl -XGET https://registry.npmjs.org/@google-cloud%2Fpubsub | json versions.'["0.28.1"]'`

I don't know off the top of my head if there is a way to do this with the `npm` bin; worth noting that a lot of npm commands accept `--json`, so if you do find a command that works you can use this to output an easily parsed value -- I love the tool [json](https://www.npmjs.com/package/json) for then facilitating transformations on the JSON data.